src/style.css

```diff
@@ -1 +1,114 @@
-/* add styles here */
+/* Обычное начертание (Normal) */
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: 400; /* Обычный вес */
+  font-style: normal;
+}
+
+/* Жирное начертание (Bold) */
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Heavy.ttf') format('truetype');
+  font-weight: 700; /* Жирный вес */
+  font-style: normal;
+}
+
+* {
+  border: none;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
```

Comment:
The `font-weight` is set to 300 in the `body` selector, but this weight is not defined in the `@font-face` rules. Consider using a defined weight like 400 or 700.